src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs

```diff
@@ -52,13 +52,17 @@
    (fn [{:keys [props state]}]
      (swap! state assoc :loading? true)
      (endpoints/call-ajax-orch
-      {:endpoint (endpoints/import-entities (:workspace-id props))
-       :raw-data (utils/generate-form-data {:entities (:file @state)})
+      {:endpoint (if (= "data" (:import-type props))
+                   (endpoints/import-entities (:workspace-id props))
+                   (endpoints/import-attributes (:workspace-id props)))
```

Comment:
Another option:
```
{:endpoint ((if (= "data" (:import-type props))
              endpoints/import-entities
              endpoints/import-attributes)
              (:workspace-id props))
```
Don't use that unless you understand it. Concision is not the highest priority.